src/ClimaLSM.jl

```diff
@@ -115,17 +116,32 @@ function initialize_interactions(land::AbstractLandModel, land_coords)
     return NamedTuple{vars}(interactions)
 end
 
-function make_ode_function(land::AbstractLandModel)
+function make_imp_tendency(land::AbstractLandModel)
     components = land_components(land)
```

Comment:
I'm not sure this would work since `typeof.(components) .<: AbstractImExModel` will return a `BitVector` of 1/0 values for each element in `components`, which can't be used in an if statement. Maybe we could instead have a for loop that goes over each component? The goal with this is to avoid unnecessary `update_aux` calls right?

I also just realized that we can't call `make_compute_imp_tendency` on an `AbstractExpModel` since it's not defined, so we'll need to change how we construct `compute_imp_tendency_list` in case `land` contains an `AbstractExpModel`.

Reply:
let's discuss in person today!

Reply:
Kat and I discussed offline - we'll do something like the following
```
if all(c -> typeof(c) .<: AbstractExpModel, components)
    return nothing
else
    do the existing code
end
```

To make sure that we are able to handle the mixed implicit/explicit case, we'll extend the `make_compute_imp_tendency` function for the `AbstractExpModel` argument type, and define this method to return a function which does nothing.